js/spec/nested-file-test.js

```diff
@@ -0,0 +1,73 @@
+const path = require('path');
```

Comment:
Please don't separate this one test out from the rest of the tests. Just inline it like the rest of the test suite please.

Reply:
Ok.